steps/property-inspection/garden.tsx

```diff
@@ -107,7 +111,11 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "property"> = {
           props: {
             name: "has-garden",
             legend: (
-              <FieldsetLegend>{questions["has-garden"]}</FieldsetLegend>
+              <FieldsetLegend>
+                <Heading level={HeadingLevels.H2}>
+                  {questions["has-garden"]}
+                </Heading>
+              </FieldsetLegend>
```

Comment:
So while trying to figure out the spacing issue I looked at other pages in our app, and found some that add a `Heading` component within the `FieldsetLegend` so I did the same...